### ISSUE TITLE
Can I display the precision, recall, mAP50, and mAP50:95 for each epochs during training?

### ISSUE THREAD

**Issue by FONG1234** [meituan/YOLOv6#806]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

This is because I want to compare my result with YOLOv5, YOLOv6, YOLOv7, and YOLOv8. However, YOLOv6 do not provide precision and recall during each epochs during training. Besides, It also do not provided the loss function curve, precision curve, recall curve, like the others YOLO version. 

Can anybody help me on this issue? I want the model to display precision and recall during each epochs when in the training phase.

**Comment by shensheng272** [comment]
Refer to [example config](https://github.com/meituan/YOLOv6/blob/main/configs/experiment/yolov6n_with_eval_params.py) to set eval params when training.
Add `--eval-interval` to train script to eval each epochs when in the training phase.
